src/client/content_cao.cpp

```diff
@@ -1561,13 +1587,9 @@ void GenericCAO::processMessage(const std::string &data)
 		}
 	} else if (cmd == GENERIC_CMD_SPAWN_INFANT) {
 		u16 child_id = readU16(is);
-		u8 type = readU8(is);
+		//u8 type = readU8(is); maybe this will be useful later
```

Comment:
since this <i>does</i> exist in the cmd, I'd prefer to read it and use `(void)` to silence the unused warning